docs/data-sources/autoscaling_group.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: "ionoscloud"
+page_title: "IonosCloud : autoscaling_group"
+sidebar_current: "docs-datasource-autoscaling_group"
+description: |-
+  Get information on a IonosCloud Autoscaling Group
+---
+
+# ionoscloud\_autoscaling_group
+
+The autoscaling group data source can be used to search for and return an existing Autoscaling Group. You can provide a string for the name or id parameters which will be compared with provisioned Autoscaling Groups. If a single match is found, it will be returned. If your search results in multiple matches, an error will be generated. When this happens, please refine your search string so that it is specific enough to return only one result.
+
+## Example Usage
+
+```hcl
+data "ionoscloud_autoscaling_group" "autoscaling_group" {
+  name = "test_ds"
+}
+```
+
+## Argument Reference
+
+* `id` - (Optional) Id of an existing Autoscaling Group that you want to search for.
+* `name` - (Optional) Name of an existing Autoscaling Group that you want to search for.
+
+Either `name` or `id` must be provided. If none or both are provided, the datasource will return an error.
+
+## Attributes Reference
+
+The following attributes are returned by the datasource:
+
+
+* `id` - UUID of the Autoscaling Group.
+* `name` - The name of the Autoscaling Group.
+* `datacenter` - VMs for this Autoscaling Group will be created in this Virtual Datacenter. Please note, that it have the same `location` as the `template`.
+    * `href` - Absolute URL to the resource's representation
+    * `type` - Type of resource
+    * `id` - Unique identifier for the resource
+* `location` - Location of the datacenter. This location is the same as the one from the selected template.
+* `max_replica_count` - Maximum replica count value for `targetReplicaCount`. Will be enforced for both automatic and manual changes.
+* `min_replica_count` - Minimum replica count value for `targetReplicaCount`. Will be enforced for both automatic and manual changes.
+* `policy` - Specifies the behavior of this Autoscaling Group. A policy consists of Triggers and Actions, whereby an Action is some kind of automated behavior, and a Trigger is defined by the circumstances under which the Action is triggered. Currently, two separate Actions, namely Scaling In and Out are supported, triggered through Thresholds defined on a given Metric.
+    * `metric` - The Metric that should trigger Scaling Actions. The values of the Metric are checked in fixed intervals.
+    * `range` - Defines the range of time from which samples will be aggregated. Default is 120s.
+      *Note that when you set it to values like 5m the API will automatically transform it in PT5M, so the plan will show you a diff in state that should be ignored.*
```

Comment:
@cristiGuranIonos I don't understand this, is it still valid? Not sure what `PT5m` means.